Storage/SQL/RemoteTabsTable.swift

```diff
@@ -20,6 +20,7 @@ class RemoteClientsTable<T>: GenericTable<RemoteClient> {
             "type TEXT",
             "formfactor TEXT",
             "os TEXT",
+            "fxaDeviceId TEXT",
```

Comment:
This is a `GenericTable`, so you'll need to override `updateTable(db: SQLiteDBConnection, from: Int)` and add an `ALTER TABLE` to add the column, and bump the `version` var at the top of this class.

Then test the shit out of it.